RELEASE.md

```diff
@@ -1,5 +1,6 @@
 ## Releasing a new version of Leaflet
 
+- [ ] Ensure no [blocker](https://github.com/Leaflet/Leaflet/labels/blocker) issues and pull requests are unresolved.
```

Comment:
> are unresolved

Maybe this should say "remain unresolved"?

Or maybe the sentence should rather say "Ensure all blocker issues and pull requests are resolved"?

Note that I refrained from using the words "open" or "closed", because that could technically indicate that closing them just to release is acceptable, and while I think everyone understands that it wouldn't be acceptable I think it's good to avoid any such indicators.

Reply:
Let us use `"Ensure all blocker issues and pull requests are resolved"`

Do we want that the label is rendered? Does it work in a .md file?
- [blocker](https://github.com/Leaflet/Leaflet/labels/blocker) `[blocker](https://github.com/Leaflet/Leaflet/labels/blocker)`
- https://github.com/Leaflet/Leaflet/labels/blocker `https://github.com/Leaflet/Leaflet/labels/blocker`

Reply:
> Let us use `"Ensure all blocker issues and pull requests are resolved"`

Agree, done!

> Do we want that the label is rendered? Does it work in a .md file?

I can confirm it works in .md files. Changed to that, thanks! Looks much prettier 🙂

Reply:
> I can confirm it works in .md files. Changed to that, thanks! Looks much prettier 🙂

Are you sure? 
![grafik](https://user-images.githubusercontent.com/19800037/155346218-fb7bae0e-cae0-4d55-acaa-cd9db7c0e9f4.png)

Reply:
@Falke-Design Yes I tested in a private repo before committing the changes.

Preview test from editing directly:

![blocker-label](https://user-images.githubusercontent.com/26493779/155347009-1c013702-9c21-45c6-b5c3-6a37175b2f87.png)

Reply:
But when you open it in the "default" way it is not visible: https://github.com/Malvoz/Leaflet/blob/release-blocker-check/RELEASE.md

Reply:
In my repo I do not have a `blocker` label: https://github.com/Malvoz/Leaflet/labels